tests/Test.ADAL.WinRT.Unit/Package.appxmanifest

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Package xmlns="http://schemas.microsoft.com/appx/2010/manifest" xmlns:m2="http://schemas.microsoft.com/appx/2013/manifest">
-  <Identity Name="5a240727-0a1a-4fda-8875-9261c8c2a35b" Publisher="CN=afshins" Version="1.0.0.0" />
```

Comment:
Is there some way to tell Visual Studio to just pick up whatever cert it sees instead of having to specify your alias?